etc/org.opencastproject.organization-mh_default_org.cfg

```diff
@@ -167,210 +167,3 @@ prop.org.opencastproject.admin.mediamodule.url=${prop.org.opencastproject.engage
-#  Start Time: Current time rounded up to the next available timeslot, or the first available timeslot on the next day.
-#  Duration:   The default duration of events
-#  End Time:   Start Time + Duration (implicit end date may be on next day)
-
-# Start time of first timeslot in the day
-# Format: HH:mm
-# Default: 08:00
-#prop.admin.event.new.start_time=08:00
-
-# Cut-off time for recordings in the day
-# Format: HH:mm
-# Default: 20:00
-#prop.admin.event.new.end_time=20:00
-
-# Duration of event recording in minutes
-# Default: 55
-#prop.admin.event.new.duration=55
-
-# Interval between recording start times in minutes
-# Default: 60
-#prop.admin.event.new.interval=60
-
-# Note: prop.admin.event.new.duration need not match prop.admin.event.new.interval;
-#       this is to allow short intermissions between consecutive events.
-
-# Preselect an appropriate weekday when scheduling multiple events.
-#
-# Format: Boolean
-# Default: true
-#
-#prop.admin.event.new.preselect_day=true
-
-#
-# Display durations for different notification types in SECONDS.
-#
-# If you specify the value -1 the notification will remain until manually closed by the user.
-
-# Display duration for notification type error
-#
-# Format: A integer or -1
-# Default: -1
-#
-#prop.admin.notification.duration.error=-1
-
-# Display duration for notification type success
-#
-# Format: A integer or -1
-# Default: 5
-#
-#prop.admin.notification.duration.success=5
-
-# Display duration for notification type warning
-#
-# Format: A integer or -1
-# Default: 5
-#
-#prop.admin.notification.duration.warning=5
-
-# Display position for global notifications
-#
-# Possible positions are:
-# - top-right
-# - top-left
-# - top-center
-# - bottom-right
-# - bottom-left
-# - bottom-center
-#
-# Default: bottom-right
-#
-#prop.admin.notification.position.global=bottom-right
-
-# If the preview mode in the video editor is enabled per default.
-#
-# Format: Boolean
-# Default: true
-#
-#prop.admin.editor.previewmode.default=true
-
-# Event access control update mode when modifying series permissions.
-# Possible modes are:
-# - always:   When modifying series permissions, automatically remove all permission rules specific to single episodes
-#             belonging to the series. This enforces every episode has the rules of the series in effect as soon as they
-#             are changed.
-# - never:    Only update the series permissions but never replace permissions set on event level. This can mean that
-#             updated rules have no effect on already existing events.
-# - optional: Like `never` but present users with a button in the series permission dialog which allows them to
-#             replace the event specific permissions if they want to.
-# Default: optional
-#prop.admin.series.acl.event.update.mode=optional
-
-# Whether the ACL of a new event is initialized with the ACL of its series.
-#
-# Format: Boolean
-# Default: true
-#prop.admin.init.event.acl.with.series.acl=true
```

Comment:
This property is still used by the new Admin UI.

Reply:
You are right. That has been recently added, but was never moved from the deprecated section in the Opencast configs.